designsafe/sitemaps.py

```diff
@@ -184,3 +201,14 @@ def items(self):
 
     def location(self, item):
         return item
+
+
+class CMSSitemap_modified(CMSSitemap):
```

Comment:
How about
```suggestion
class DesignSafeCMSSitemap(CMSSitemap):
```
?

Reply:
Sure! I'll need to change that name in a couple places.

Reply:
done.